save-cosv/src/main/kotlin/com/saveourtool/save/cosv/controllers/RawCosvFileController.kt

```diff
@@ -386,6 +372,34 @@ class RawCosvFileController(
                 }
         }
 
+    /**
+     * @param organizationName
+     * @param authentication
+     * @return [StringResponse]
+     */
+    @RequiresAuthorizationSourceHeader
+    @DeleteMapping("/delete-all-duplicated-files")
+    fun deleteAllDuplicatedFiles(
+        @PathVariable organizationName: String,
+        authentication: Authentication,
+    ): Mono<StringResponse> = rawCosvFileStorage.listByOrganizationAndUser(organizationName, authentication.name)
+        .map { files ->
+            files.filter { it.status == RawCosvFileStatus.FAILED && it.statusMessage?.contains("Duplicate entry") == true }.map { it.requiredId() }
```

Comment:
please extract `private fun RawCosvFileDto.isDuplicate() = status == RawCosvFileStatus.FAILED && statusMessage?.contains("Duplicate entry") == true` to common place (`RawCosvFileDto.kt` for example) and re-use here and there